src/openforms/celery.py

```diff
@@ -17,3 +24,41 @@
 }
 
 app.autodiscover_tasks()
+
+
+#
+# Utilities for checking the health of celery workers
+#
+class LivenessProbe(bootsteps.StartStopStep):
+    requires = {"celery.worker.components:Timer"}
+
+    def __init__(self, worker, **kwargs):
```

Comment:
I'm not sure how to get code coverage to pick up these lines. But to ignore them, you can use the `# noqa` comment